docs/4.0. Behaviour.md

```diff
@@ -23,3 +23,12 @@ In environments where multiple clients may be operating against a single Connect
 ## 'Salvo' Operation
 
 Where a server implementation supports concurrent application of settings changes to underlying Senders and Receivers, it may choose to perform 'bulk' resource operations in a parallel fashion internally. This is an implementation decision and is not a requirement of this specification.
+
+# Scheduled Activations
+
+IS-05 provides a mechanism whereby the activation of staged transport parameters can be performed at a particular TAI time, or after a given interval.
+The primary use case for this behaviour is to allow the synchronisation of large salvo operations, where multiple pieces of equipment should carry out a change in transport parameters simultaneously. A controller may stage parameters with multiple IS-05 APIs with the same activation timestamp, and know that activation will occur at the same time on all devices regardless of the latency between the controller and the device. It is not intended to act as a mechanism for scheduling activations far in the future.
+
+In the event of an error occurring between scheduling an activation and the activation time, the IS-05 transport parameters should reflect the current state of the device. For example, if a sender is no longer sending data at all it should reflect this by setting `rtp_enabled` to false.
+
+API clients should check back with the API after the expected activation time to ensure activations have completed successfully, and that the device is in the expected state. Note that a client may normally expect to see a change in the IS-04 version timestamp upon a successful update of transport parameters, and should monitor for this version timestamp update via the IS-04 web-socket.
```

Comment:
Perhaps explicitly say here when referencing IS-04 'if IS-04 is in use alongside IS-05'.

Reply:
Agree with Andrew.